eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -131,16 +131,18 @@ public static function installWelcomeText(Event $event)
 
 <fg=cyan>Welcome to ${installName}!</fg=cyan>
 
-<options=bold>Quick dev/test install:</>
+<options=bold>Quick installation to test in local dev environment:</>
 <comment>    $  export SYMFONY_ENV="dev"</comment>
 ${installCommandText}
 <comment>    $  php ${consoleDir}/console server:run</comment>
 
 Note:
-- Instructions above assume the CLI user you execute these commands with is the same one that extracted/installed the software.
-- The last command will give you the url to the frontend of the installation, add "/admin" to reach backend.
+- The instructions assume you execute commands with the CLI user that extracted/installed the software.
+- The "server:run" command will:
+  - Use PHP's single process, local, HTTP/1 only built-in web server, mainly suitable for testing.
+  - Give you the URL to the front end of the installation. TIP: Add "/admin" to reach back end.
 
-For full install instructions, both for production and better performing dev setup, see:
+For full install instructions with Nginx/Apache for production, remote use, or better performing dev setup:
```

Comment:
Maybe even this, to get rid of the hanging "for":
```suggestion
See main installation instructions with Nginx/Apache for production, remote use, or better performing dev setup in:
```
But both versions are fine by me.